typescript/phoenix-sdk/src/events.ts

```diff
@@ -101,7 +102,7 @@ export async function getEventsFromTransaction(
           tradeEvents.push({
             index: reader.readU16(),
             makerId: readPublicKey(reader),
-            orderSequenceNumber: reader.readFixedArray(8),
+            orderSequenceNumber: reader.readU64(),
```

Comment:
I think we probably want to read this in as an `i64`. Seems a little hacky, but the sign will indicate which side of the book the order is on